app/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt

```diff
@@ -53,25 +55,44 @@ interface VariantManager {
             Variant(
                 key = "me",
                 weight = 0.0,
-                features = listOf(ConceptTest, SuppressWidgetCta, SuppressDefaultBrowserCta),
+                features = listOf(ConceptTest, SuppressHomeTabWidgetCta, SuppressOnboardingDefaultBrowserCta),
+                filterBy = { isEnglishLocale() }),
+            Variant(
+                key = "md",
+                weight = 0.0,
+                features = listOf(SuppressHomeTabWidgetCta, SuppressOnboardingDefaultBrowserCta),
                 filterBy = { isEnglishLocale() }),
-            Variant(key = "md", weight = 0.0, features = listOf(SuppressWidgetCta, SuppressDefaultBrowserCta), filterBy = { isEnglishLocale() }),
 
             // Validate CTAs experiment
             Variant(key = "mq", weight = 0.0, features = emptyList(), filterBy = { noFilter() }),
-            Variant(key = "mr", weight = 0.0, features = listOf(SuppressDefaultBrowserCta), filterBy = { noFilter() }),
-            Variant(key = "ms", weight = 0.0, features = listOf(SuppressWidgetCta), filterBy = { noFilter() }),
-            Variant(key = "mt", weight = 0.0, features = listOf(SuppressWidgetCta, SuppressDefaultBrowserCta), filterBy = { noFilter() }),
+            Variant(key = "mr", weight = 0.0, features = listOf(SuppressOnboardingDefaultBrowserCta), filterBy = { noFilter() }),
+            Variant(key = "ms", weight = 0.0, features = listOf(SuppressHomeTabWidgetCta), filterBy = { noFilter() }),
+            Variant(
+                key = "mt",
+                weight = 0.0,
+                features = listOf(SuppressHomeTabWidgetCta, SuppressOnboardingDefaultBrowserCta),
+                filterBy = { noFilter() }),
 
             // Insert CTAs on Concept test experiment
```

Comment:
@cmonfortep if you are running new CTA experiments you need to refresh the variants otherwise you'll be comparing some users who are on the latest version of the app e.g "mh" to other users who span multiple versions e.g the preexisting "mu", "mv"m "mz".

Reply:
Latest update resolves this.